packages/dds/map/src/directory.ts

```diff
@@ -621,6 +629,8 @@ export class SharedDirectory
 					let newSubDir = currentSubDir.getSubDirectory(subdirName) as SubDirectory;
 					if (!newSubDir) {
 						newSubDir = new SubDirectory(
+							0,
```

Comment:
is this really safe? I'm imagining a scenario like this might be problematic:

- clients 1, 2, 3 exist in session
- seq 10: client 1 creates directory "foo"
- client 3 submits a property change on "foo" with refSeq 10 but has high latency (op isn't acked)
- seq 11: client 2 deletes "foo"
- seq 12: client 2 creates "foo"
- summarization occurs. minSeq is 10, refSeq for the summary is 12
- client 4 joins the session, loading that summary. it populates "foo"
- client 3's op is finally acked.

the danger here would be: client 1, 2, and 3 recognize that client 3's op was meant for a different instance of the "foo" directory and so wouldn't apply it. client 4 wouldn't realize this and so would apply it.

i'd expect some kind of summary format change or strategy along the lines of "preserve information within the collab window" to be necessary.

Reply:
Mage the change.

Reply:
cool, i"ll take a look